packages/auth/test/integration/webdriver/redirect.test.ts

```diff
@@ -232,7 +232,7 @@ browserDescribe('WebDriver redirect IdP test', driver => {
       'user@test.test'
     );
 
-    // Link using pre-poulated user
+    // Link using pre-populated user
```

Comment:
If I make updates to this again, I'm likely to change it to `prepopulated` (dropping the `-`)